jscomp/test/event_ffi.js

```diff
@@ -30,12 +30,11 @@ function h33(x) {
 }
 
 function h34(x) {
-  return x(1, 2, 3)(4);
+  return x(1, 2, 3, 4);
```

Comment:
This doesn't look correct.

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/6904